mintpy/plot_network.py

```diff
@@ -76,6 +77,19 @@ def create_parser():
     coh.add_argument('-v', '--vlim', nargs=2, type=float, default=(0.2, 1.0),
                      help='display range')
 
+    # Display baseline in the network plot
+    coh = parser.add_argument_group('Display Baseline', 'Color lines in the network plot with baseline instead of coherence')
```

Comment:
`coh` name is taken for the previous argument group, let's rename it to something else, e.g. `base`?